src/aiida_dftk/calculations.py

```diff
@@ -4,15 +4,18 @@
 import os
 import json
 import typing as ty
-from pathlib import Path
 
 from aiida import orm
 from aiida.common import datastructures, exceptions
-from aiida.engine import CalcJob
+from aiida.engine import CalcJob, ExitCode
 from aiida_pseudo.data.pseudo import UpfData
 from pymatgen.core import units
 
 
+_AIIDA_DFTK_MIN_VERSION = "0.1.9" # inclusive
+_AIIDA_DFTK_MAX_VERSION = "0.2" # exclusive
```

Comment:
As discussed on julia side: Switch to single version string and Pkg.jl conventions if possible.